apps/patient/src/utils/isGLP.ts

```diff
@@ -0,0 +1,26 @@
+const commonGLPs = [
+  'ozempic',
+  'rybelsus',
+  'wegovy',
+  'trulicity',
+  'victoza',
+  'saxenda',
+  'byetta',
+  'bydureon',
+  'mounjaro',
+  'zepbound',
+  'adlyxin',
+  'tanzeum',
+  'semaglutide',
+  'liraglutide',
+  'dulaglutide',
+  'exenatide',
+  'tirzepatide',
+  'lixisenatide',
+  'albiglutide'
+];
+
+export function isGLP(name: string) {
+  const regex = new RegExp(commonGLPs.join('|'), 'i');
```

Comment:
the `i` lowercases right?

Reply:
yup!

Reply:
https://www.tutorialspoint.com/How-to-perform-Case-Insensitive-matching-with-JavaScript-RegExp